specs/OracleQueries.yml

```diff
@@ -246,6 +246,21 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/registerLocation'
+  /company/{companyNumber}/update/entityData:
+    parameters:
+      - $ref: '#/components/parameters/companyNumber'
+    get:
+      tags:
+        - update-entity-data
+      operationId: getEntityData
+      summary: Get private Entity data during Update flow
+      responses:
+        '200':
+          description: 'Retrieve Entity data'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/entityData'
```

Comment:
Do we need new response schema? entityData sounds like something else but have not looked into schema defs below

Reply:
overseasEntityData?